apps/dotcom/client/src/components/MultiplayerEditor.tsx

```diff
@@ -121,11 +122,16 @@ export function MultiplayerEditor({
 }) {
 	// make sure this runs before the editor is instantiated
 	useLegacyUrlParams()
+	const user = useLoggedInUser()
 
 	const handleUiEvent = useHandleUiEvents()
 
 	const storeWithStatus = useSync({
-		uri: `${MULTIPLAYER_SERVER}/${RoomOpenModeToPath[roomOpenMode]}/${roomSlug}`,
+		uri: useCallback(
+			async () =>
+				`${MULTIPLAYER_SERVER}/${RoomOpenModeToPath[roomOpenMode]}/${roomSlug}?clerk_token=${await user.getToken()}`,
+			[roomOpenMode, roomSlug, user]
+		),
```

Comment:
i think we should undo the change in here, `MultiplayerEditor` is used on current dotcom and we should leave it alone and do everything with the `TlaEditor` component.